### PR TITLE
Add local storage db

### PR DESCRIPTION
**Pull request by andrecardoso**
Most of this code is being ported from `local-storage-module`:

- Handle folder watch
- Add lokijs database
- Add file expiration handling

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(3) ChrOS: in-memory DB (or Map) for maintaining watch list](https://trello.com/c/bRbkmZ4Y/4097-3-chros-in-memory-db-or-map-for-maintaining-watch-list)

File: src/storage/database/api.js [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        try {
+          watchlist.update(item);
+        } catch (err) {
+          rej(err);
+        }
+
+        res();
+      });
+    },
+    runtimeInfo() {
```
Comment (tejohnso) [inline src/storage/database/api.js]: Can you explain what all is going on with runtimeInfo runtimeSequence?

Comment (andrecardoso) [reply src/storage/database/api.js]: It's a copy of this feature to auto expire entries: https://github.com/Rise-Vision/local-storage-module/pull/89

**Comment by andrecardoso** [comment]
[(5) ChrOS: Handle WATCH, WATCH-RESULT for folders](https://trello.com/c/iTMb10Gk/4141-5-chros-handle-watch-watch-result-for-folders)

**Pull request merged**